server/game/game.js

```diff
@@ -102,11 +102,17 @@ class Game extends EventEmitter {
     }
 
     getPlayers() {
-        return _.omit(this.playersAndSpectators, player => this.isSpectator(player));
+        return Object.values(this.playersAndSpectators).filter(player => !this.isSpectator(player));
```

Comment:
A little less succinct than _.filter() (which this should have been..) but also removes the ambiguity of arrays/objects one sometimes gets into when working with underscore, so on the whole i think i like it 👍